explore-analyze/dashboards/add-controls.md

```diff
@@ -199,3 +201,179 @@ Delete controls from your dashboard.
 :::
+
+::::
+______________________________
+::::{applies-item} stack: ga 9.2
+## Create and add Options list and Range slider controls [create-and-add-options-list-and-range-slider-controls]
+
+To add interactive Options list and Range slider controls, create the controls, then add them to your dashboard.
+
+1. Open or create a new dashboard.
+2. In **Edit** mode, select **Add** > **Controls** > **Control** in the toolbar.
+3. On the **Create control** flyout, from the **Data view** dropdown, select the data view that contains the field you want to use for the **Control**.
+4. In the **Field** list, select the field you want to filter on.
+5. Under **Control type**, select whether the control should be an **Options list** or a **Range slider**.
+   ::::{tip}
+   Range sliders are for Number type fields only.
+   ::::
+
+6. Define how you want the control to appear:
+
+    * **Label**: Overwrite the default field name with a clearer and self-explanatory label.
+    * **Minimum width**: How much horizontal space does the control occupies. The final width can vary depending on the other controls and their own width setting.
+    * **Expand width to fit available space**: Expand the width of the control to fit the available horizontal space on the dashboard.
+
+7. Specify the additional settings:
+
+    * For Options lists:
+
+        * Define whether users can select multiple values to filter with the control, or only one.
+        * For Options list controls on *string* and *IP address* type fields, you can define how the control’s embedded search should behave:
+
+            * **Prefix**: Show options that *start with* the entered value.
+            * **Contains**: Show options that *contain* the entered value. This setting option is only available for *string* type fields. Results can take longer to show with this option.
+            * **Exact**: Show options that are a 100% match with the entered value.
+
+              ::::{tip}
+              The search is not case sensitive. For example, searching for `ios` would still retrieve `iOS` if that value exists.
+              ::::
+
+        * **Ignore timeout for results** delays the display of the list of values to when it is fully loaded. This option is useful for large data sets, to avoid missing some available options in case they take longer to load and appear when using the control.
+
+    * For Range sliders, set the step size. The step size determines the number of steps of the slider. The smaller the step size is, the more steps there is in the slider.
+
+8. Select **Save**. The control can now be used.
+9. Consider the position of the control if you have several controls active on the dashboard. Controls are applied from left to right, which can change the options available depending on their position when the [Chain controls](#configure-controls-settings) setting is enabled.
```

Comment:
```suggestion
9. Consider the position of the control if you have several controls active on the dashboard. When the [Chain controls](#configure-controls-settings) setting is enabled, controls are applied from left to right, which can change the available options based on their position.
```
It wasn't clear if the left-to-right behavior was only applicable when chain controls is enabled. But I think that's the case, right?